api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -236,6 +256,12 @@ message AccessLogCommon {
   // For HTTP: Total number of bytes received from the upstream by the http stream.
   // For TCP: Total number of bytes sent to the upstream by the tcp proxy.
   uint64 upstream_wire_bytes_received = 32;
+
+  // The type of the access log, which indicates when the log was recorded.
+  // See :ref:`ACCESS_LOG_TYPE <config_access_log_format_access_log_type>` for the available values.
+  // In case the access log was recorded by a flow which does not correspond to one of the supported
+  // values, then an empty string will be the default value.
```

Comment:
I think the default value should be `NotSet`. And I think similar contraints are necessary, for example, the start/end log for every stream should be outputed for only once.

Reply:
Added reference to access log docs as they are more detailed instead of trying to summarize this here